indra/newview/skins/default/xui/en/notifications.xml

```diff
@@ -10348,6 +10348,14 @@ You are now the owner of object [OBJECT_NAME]
 
   <notification
    icon="alertmodal.tga"
+   name="NowOwnObjectInv"
+   type="notify">
+   <tag>fail</tag>
```

Comment:
probably shouldn't be 'fail'

Reply:
All the others around use fail... (It's supposed to behave like `NowOwnObject` immediately above it and that is marked as fail also.